jobs/src/main/scala/dcos/metronome/scheduler/SchedulerModule.scala

```diff
@@ -88,22 +88,9 @@ class SchedulerModule(
     leadershipModule,
     () => scheduler.getLocalRegion)(actorsModule.materializer)
 
-  private[this] lazy val offerMatcherReconcilerModule =
-    new OfferMatcherReconciliationModule(
-      scallopConf,
-      clock,
-      actorSystem.eventStream,
-      instanceTrackerModule.instanceTracker,
-      persistenceModule.groupRepository,
-      leadershipModule)(actorsModule.materializer)
-
-  lazy val taskStatusProcessor: TaskStatusUpdateProcessor = new TaskStatusUpdateProcessorImpl(
-    metrics, clock, instanceTrackerModule.instanceTracker, schedulerDriverHolder, killService, eventBus)
-
   private[this] lazy val launcherModule: LauncherModule = {
     val instanceTracker: InstanceTracker = instanceTrackerModule.instanceTracker
     val offerMatcher: OfferMatcher = StopOnFirstMatchingOfferMatcher(
```

Comment:
Take a look at `StopOnFirstMatchingOfferMatcher`: you don't even need to use that one and could just do
```
val offerMatcher: OfferMatcher = offerMatcherManagerModule.globalOfferMatcher
```

:D

Reply:
LOL :) You're right as always :)